src/topology/algebra/const_mul_action.lean

```diff
@@ -45,6 +45,9 @@ local attribute [instance] mul_action.orbit_rel
 /-- Class `has_continuous_const_smul Γ T` says that the scalar multiplication `(•) : Γ → T → T`
 is continuous in the second argument. We use the same class for all kinds of multiplicative
 actions, including (semi)modules and algebras.
+
+Note that both `has_continuous_const_smul α α` and `has_continuous_const_smul αᵐᵒᵖ α` are
+generalisations of `has_continuous_mul α`.
```

Comment:
Can you add something similar to `has_continuous_mul` too? Perhaps something like:

> For cases where multiplication is only continuous in the left or right argument separately, `has_continuous_const_smul α α` or `has_continuous_const_smul αᵐᵒᵖ α` can be used, respectively.